app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    memorized_function_results = {}
+
+    def wrapper(*args):
+        if args not in memorized_function_results:
+            print("Calculating new result")
+            memorized_function_results[args] = func(*args)
+            return memorized_function_results[args]
```

Comment:
well, you have 2 the same `return memorized_function_results[args]` - make only one return